src/game/game2048.ts

```diff
@@ -29,7 +29,7 @@ export interface IGameState {
   private scores: number = 0;
   private grid: Grid;
   private userActionsQueue: Action[] = [];
-
+  public highScore: number;
```

Comment:
We're start storing this state between game sessions

Reply:
@davidkwlam What do you think? Maybe @matthewjamesadam too?

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/c173f49b-bc4a-433b-acc4-d279ef5cd711?message=90bb731b-99ba-413f-a421-036a6d04b6da).

Reply:
Agree, let's make that change

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/c173f49b-bc4a-433b-acc4-d279ef5cd711?message=186c1161-b7d0-4ba1-9a25-5abccc6b89f3).

Reply:
Yeah that sounds good to me too

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/c173f49b-bc4a-433b-acc4-d279ef5cd711?message=c13566c4-2d09-475c-83be-776878ed63fd).

Reply:
Let's do it!

Reply:
What's the delay?

Reply:
I'm slow, haven't had coffee!

--

Comment posted using [Unblocked](https://getunblocked.com/dashboard/team/461c33ed-2d8d-468d-9409-d2cec26e4741/thread/c173f49b-bc4a-433b-acc4-d279ef5cd711?message=2c10c88e-458f-4efb-aa37-3dc175815e56).